src/components/app-configure/AppAuth.vue

```diff
@@ -76,48 +68,55 @@
             placeholder="Enter Redirect Url"
             v-model="selectedAuthRedirectUrl"
           />
-          <v-button
-            variant="secondary"
-            @click.stop="addAuthentication"
-            v-if="selectedAuthenticationType"
-            label="ADD"
-          />
+          <div v-if="selectedAuthenticationType" style="margin-top: 1em">
+            <v-tooltip title="Add">
+              <v-icon-button
+                :icon="CheckIcon"
+                style="margin-left: 1em"
+                @click.stop="addAuthentication"
+              />
+            </v-tooltip>
+            <v-tooltip title="Clear">
+              <v-icon-button
+                :icon="CancelIcon"
+                style="margin-left: 1em"
+                @click.stop="clearAuthentication"
+              />
+            </v-tooltip>
+          </div>
         </div>
         <div
           v-if="selectedAuthenticationType"
-          style="
-            overflow-x: hidden;
-            text-overflow: ellipsis;
-            margin-bottom: 1em;
-          "
+          style="overflow-x: hidden; text-overflow: ellipsis"
         >
-          <span class="body-2" style="line-height: 2em">
+          <span class="body-2" style="line-height: 1.5em">
```

Comment:
Line heights are unitless.

```suggestion
          <span class="body-2" style="line-height: 1.5">
```